package.json

```diff
@@ -28,6 +28,7 @@
   },
   "scripts": {
     "bootstrap": "lerna bootstrap",
+    "postinstall": "lerna bootstrap --hoist",
```

Comment:
Because a few reason we don't do this, and instead bootstrap manually.

1. For some CI tasks bootstrapping is unnecessary, but installing is. Bootstrapping is rather slow.
2. We do npm install in examples, this will cause the postinstall to re-run, causing a lot of unnecessary delay.
3. When adding a root dependency for development, bootstrapping is not needed.